conu/backend/docker/container.py

```diff
@@ -370,3 +370,14 @@ def exit_code(self):
         :return: int
         """
         return self.get_metadata()["State"]["ExitCode"]
+
+    def active_write(self, message):
+        """
+        Send a message to running container
+
+        :param message: input to be written on container terminal
+        """
+        if not self.is_running():
+            raise ConuException("Container must be running")
+        self.popen_instance.stdin.write(message)
```

Comment:
And we should also probably catch exceptions from the write and reraise using our exception.